saul-core/src/test/scala/JoinTrainingTests/intializeSparseNetwork.scala

```diff
@@ -0,0 +1,90 @@
+package JoinTrainingTests
+
+import edu.illinois.cs.cogcomp.infer.ilp.OJalgoHook
+import edu.illinois.cs.cogcomp.lbjava.infer.FirstOrderConstant
+import edu.illinois.cs.cogcomp.lbjava.learn.{ LinearThresholdUnit, SparseNetworkLearner }
+import edu.illinois.cs.cogcomp.saul.classifier.{ JointTrainSparseNetwork, ClassifierUtils, ConstrainedClassifier, Learnable }
+import edu.illinois.cs.cogcomp.saul.datamodel.DataModel
+import org.scalatest.{ FlatSpec, Matchers }
+
+/** Created by Parisa on 9/18/16.
```

Comment:
I think we have not been consistent about it. what is the idea behind this policy?